src/components/dashboards/index.tsx

```diff
@@ -734,17 +815,21 @@ const Card: React.FC<CardProps> = React.memo(({ panel }) => {
                     />
                   )}
                   {panel.type === 'graph' && (
-                    <Graph
-                      formatSeriesTitle={formatSeriesTitle}
-                      isStack={panel.stack}
-                      pollInterval={pollInterval}
-                      queries={queries}
-                      showLegend={panel.legend?.show}
-                      units={panel.yaxes?.[0]?.format}
-                      onZoomHandle={handleZoom}
-                      namespace={namespace}
-                      customDataSource={customDataSource}
-                    />
+                    <>
+                      {queries}
```

Comment:
Are the queries needed here?

Reply:
removed queries as this was only added to see the the data that's being called on the cards in the dashboard